Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -762,7 +762,11 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.tap(app.buttons["newTabButtonTabTray"],
                         forAction: Action.OpenNewTabFromTabTray,
                         transitionTo: NewTabScreen)
-        screenState.tap(app.toolbars.buttons["closeAllTabsButtonTabTray"], to: CloseTabMenu)
+        if isTablet {
```

Comment:
I've discovered that the "Close All Tabs" button is on the top navigation bar instead of the bottom toolbars just on the iPad.